darkmoon/api/v1/metadata/views.py

```diff
@@ -1,9 +1,4 @@
-"""This is the router file."""
-
-###########
-# IMPORTS #
-###########
-
+"""This is the views.py file."""
```

Comment:
If we are keeping these docstrings at the top, they need to be more useful than stating the name of the file.